### DISCUSSION TITLE
Platforms

### DISCUSSION THREAD

**Discussion by brandonp2412** [brandonp2412/Flexify#25] [Polls]
What platforms would people like this app on in addition to Android

**Comment by civilblur** [comment]
Been looking for an app like Flexify for the longest time and I've had a pleasant experience using it for the past weeks.

I voted for "Web" as I'm hoping for a way to be able to sync the data across platforms without relying on e.g. syncthing or nextcloud. Although, I'm not sure if "Web" has this implication given the current spirit of being offline-first.

Essentially, the main feature missing for me is a selfhosted server for syncing data across platforms. The app is great during working out, but for managing data, a web interface on a desktop is preferred.

Additionally, is there a way to support the project through donations?

**Comment by brandonp2412** [comment]
Yeah have thought about syncing via some kind of LAN connection. Might add donation stuff later but nothings set up yet. If you wanted to sync between devices using CSV import/export would be the way to go. Right now if I'm doing anything complicated I export to CSV and use libreoffice personally.

**Reply by civilblur** [reply]
Excellent, excited to hear that the sync idea is at least not off the table.

**Reply by brandonp2412** [reply]
Yeah I'm just not experienced in doing anything like that could be cool to learn. Something I was going to add is an automatic export to CSV, perhaps when a plan is finished for the day or could even be on every save (run in BG thread). Then if you had Syncthing it should auto copy the db across

**Reply by civilblur** [reply]
That sounds like a good approach as a middle ground, and would likely cover most people's use cases.

**Comment by brandonp2412** [comment]
Just as an update the web support is hard to get working with Drift, i just get a bazillion errors every time I try

**Comment by brandonp2412** [comment]
Flexify now supports web https://brandonp2412.github.io/Flexify/
Since all platforms are now supported, i'm closing this ticket

**Reply by civilblur** [reply]
Very exciting, it's been a long-awaited feature for me! Will give it a test soon and report back if I happen to find any issues.
